model/labels/labels_dedupelabels.go

```diff
@@ -417,6 +417,13 @@ func (ls Labels) WithoutEmpty() Labels {
 	return ls
 }
 
+// ByteSize returns the approximate size of the labels in bytes.
+// String header size is ignored because it should be amortized to zero.
+// TODO: this is not correct yet
```

Comment:
I think this is fine - it would be confusing to take into account the size of the strings in the symbol table, if that's what you're thinking.

Reply:
Yeah I am thinking to take into account of they symbol table.

Reply:
I think what you have in this PR already is fine.